src/lib/transform-detect-infinite-loop.ts

```diff
@@ -0,0 +1,89 @@
+// Based on https://repl.it/site/blog/infinite-loops by Amjad Masad.
+
+export default function TransformDetectInfiniteLoop({ types: t }) {
+	const MAX_ITERATIONS = 2000;
```

Comment:
Suggest moving these to environment variables (or to another constants definition that's higher-level in the project).